src/lib.rs

```diff
@@ -40,6 +38,21 @@ pub const ATTEMPT_COUNTER_DEFAULT_RETRIES: u8 = 8;
 /// Do not make longer messages than this size
 pub const CTAPHID_MESSAGE_SIZE_LIMIT: usize = 3072;
 
+#[repr(u8)]
+pub enum CustomStatus {
+    /// The custom status to be shown for the successful verification for the Reverse HOTP.
+    /// By design this should be: blink green LED for 10 seconds, highest priority
+    ReverseHotpSuccess = 0,
+
+    /// The custom status to be shown for the failed verification for the Reverse HOTP.
+    /// By design this should be: blink red LED infinite times, highest priority
+    ReverseHotpError = 1,
+}
+
+/// Deny Reverse HOTP request, if required time from the last failed verification attempt has not passed yet
+/// Makes brute-force attack slower.
+pub const REQUIRED_DELAY_ON_FAILED_VERIFICATION: Duration = Duration::from_secs(5);
```

Comment:
As the error state cannot be overwritten, does it even make sense to allow subsequent verification attempts after a failed one?

Reply:
I have not considered that case before. Do you suggest instead a complete device lock (e.g. somewhat panic-like) until power-cycle is done?
The current Nitrokey Pro/Storage behavior is to allow it anyway, but the LED color status is not changing regardless of the result.

Reply:
We did. Do you want to talk about this in a separate ticket then?

Reply:
That makes sense so that we have a reference for future discussions.